app/components/buttons/DeleteButton.tsx

```diff
@@ -11,17 +12,9 @@ export default function DeleteButton({
   active: boolean;
 }) {
   return (
-    <button
-      disabled={!canDelete}
-      type="button"
-      onClick={() => setShowModal(true)}
-      className={classNames(
-        'group flex items-center px-4 py-2 text-sm w-full',
-        active && canDelete ? 'bg-gray-100 text-gray-900' : 'text-gray-700',
-      )}
-    >
+    <Button color="danger" size="sm" disabled={!canDelete} type="button" onClick={() => setShowModal(true)}>
```

Comment:
since `button` is the default type of the component `Button`, let's remove it.

Reply:
okay